ironic/tests/unit/db/utils.py

```diff
@@ -425,6 +425,15 @@ def get_test_redfish_info():
     }
 
 
+def get_test_drac_redfish_info():
+    return {
+        "redfish_address": "https://192.168.110.13",
```

Comment:
Hi Chris,
at line no: 430 to 333, if we give correct credentials then only redfish api test cases executes successfully.

Reply:
Hey Rachit, this is because the code ends up calling _get_mac_address(), which uses requests to query the MAC address from the targeted server.  When you're running the test cases, it's actually retrieving the MAC from 110.13 over the network!
I think you should remove get_test_drac_redfish_info() and use get_test_redfish_info() instead.  In your test cases that result in the MAC being queried, you will either need to mock requests.get() or mock _get_mac_address() and write more test cases to test _get_mac_address() in isolation.

Reply:
Done